src/main/config/config_streamer.c

```diff
@@ -53,6 +53,8 @@ extern uint8_t __config_end;
 #  define FLASH_PAGE_SIZE                 ((uint32_t)0x8000)
 # elif defined(UNIT_TEST)
 #  define FLASH_PAGE_SIZE                 (0x400)
+# elif defined(SITL)
```

Comment:
fwiw, i think sitl is self explanatory, and if not... first result on google...

![screen shot 2017-04-04 at 9 20 54 pm](https://cloud.githubusercontent.com/assets/399770/24689619/9d701dce-197c-11e7-8157-dfcfc3f59b53.png)